ydb/core/protos/node_whiteboard.proto

```diff
@@ -193,6 +193,7 @@ message TVDiskStateInfo {
     optional EVDiskState VDiskState = 11 [(DefaultField) = true];
     // Disk space flags
     optional EFlag DiskSpace = 12 [(DefaultField) = true];
+    optional TOosParams OosParams = 32 [(DefaultField) = true];
```

Comment:
it's a bad idea to a put blob storage internals attributes to whiteboard and making it a default - even worse.

if you want a space usage - make it in numbers, not in psychedelic rainbow.